STAR/draft-ietf-acme-star.md

```diff
@@ -478,39 +511,40 @@ illustrated in {{figunauthgetstarcert}}.
 ~~~
 {: #figunauthgetstarcert title="Fetching a STAR certificate with unauthenticated GET"}
 
-The Server SHOULD include the "Cert-Not-Before" and "Cert-Not-After" HTTP headers in the response.
-When they exist, they MUST be equal to the respective fields inside the end-entity certificate. Their format is "HTTP-date" as defined in Section 7.1.1.2 of {{RFC7231}}.
+The Server SHOULD include the "Cert-Not-Before" and "Cert-Not-After" HTTP header fields in the response.
+When they exist, they MUST be equal to the respective fields inside the end-entity certificate. Their format is "HTTP-date" as defined in Section 7.1.1.2 of {{RFC7231}}, protected with double-quotes to allow detecting and potentially recovering from situations where misbehaving intermediaries coalesce single value header fields (see Section 8.3.1 of {{RFC7231}}).
```

Comment:
As discussed, please remove the "protected with" text.

Reply:
done!